### DISCUSSION TITLE
How to remove files from Git LFS?

### DISCUSSION THREAD

**Discussion by dkobayas-cyber** [git-lfs/git-lfs#5225] [Q&A]
I'd like to downgrade the Git LFS Data, but since I'm already using up 1.94 GB of quota, I cannot do so. I'm wondering how I can remove some files from Git LFS... 

My repo listed at the top uses the largest quota. 
![image](https://user-images.githubusercontent.com/85417498/207975415-d0dccbd2-02a1-47d8-b511-b3757d5e415a.png)
`.attributes` was tracking `.mat` files, so I tried deleting some large `.mat` files. However, even after the deletion and commit, the quota usage remains to be 1.47 GB. 

As in the [Github doc - Removing files from Git Large File Storage](https://docs.github.com/en/repositories/working-with-files/managing-large-files/removing-files-from-git-large-file-storage), I tried using BFG repo-cleaner and removed the 10 largest files from the repo, but it didn't help either. My questions are: 

1. In the first place, can't I downgrade the Git LFS if the quota usage is more than what is allowed in the free LFS account? 
2. Is there any way to check which file is saved by Git LFS and delete specific files? 

I saw several similar questions, but none of them answered my questions.

**Comment by chrisd8088** [comment]
Hey, I think this is probably a question best directed to Github Support, as this project is just for the open-source Git LFS client.

My understanding is that, per this [section](https://docs.github.com/en/repositories/working-with-files/managing-large-files/removing-files-from-git-large-file-storage#git-lfs-objects-in-your-repository) of the GitHub documentation, the only way you can reduce usage is to delete the repository and recreate it, because GitHub does not prune individual Git LFS objects even if they are not reachable in your Git history any more.

**Reply by dkobayas-cyber** [reply]
> Hey, I think this is probably a question best directed to Github Support, as this project is just for the open-source Git LFS client.
> 
> My understanding is that, per this [section](https://docs.github.com/en/repositories/working-with-files/managing-large-files/removing-files-from-git-large-file-storage#git-lfs-objects-in-your-repository) of the GitHub documentation, the only way you can reduce usage is to delete the repository and recreate it, because GitHub does not prune individual Git LFS objects even if they are not reachable in your Git history any more.

Yeah, that seems like the only option. I'll try deleting the repo and see if it really fixes the problem. 

Thank you for your comment!

**Reply by Nielsbishere** [reply]
This does not seem to work for me, now I just have GBs of data that are apparently from the void
